exchanges/bithumb/bithumb.go

```diff
@@ -612,3 +615,41 @@ func (b *Bithumb) GetCandleStick(symbol, interval string) (resp OHLCVResponse, e
 	err = b.SendHTTPRequest(exchange.RestSpot, path, &resp)
 	return
 }
+
+// FetchExchangeLimits fetches spot order execution limits
+func (b *Bithumb) FetchExchangeLimits() ([]order.MinMaxLevel, error) {
+	ticks, err := b.GetAllTickers()
```

Comment:
This does highlight that we may need to be routinely calling `FetchExchangeLimits` for enabled exchanges - as a long running application, this could get stale really quickly and unable to place valid orders.

Reply:
Added to Trello: [here](https://trello.com/c/OdGZmGIH) Thanks @thrasher-